app/views/directives/process-template.html

```diff
@@ -1,46 +1,28 @@
-<div class="osc-form">
-  <alerts alerts="$ctrl.alerts"></alerts>
-  <div class="row">
-    <div class="col-md-2 icon hidden-sm hidden-xs">
-      <custom-icon resource="$ctrl.template" kind="template"></custom-icon>
+<fieldset ng-disabled="disableInputs">
+  <form name="$ctrl.templateForm">
+    <template-options is-dialog="$ctrl.isDialog" parameters="$ctrl.template.parameters" expand="true" can-toggle="false">
+      <select-project ng-if="!$ctrl.project" selected-project="$ctrl.selectedProject" name-taken="$ctrl.projectNameTaken"></select-project>
+    </template-options>
+    <!-- TODO: Also support advanced options for new projects. -->
+    <div ng-if="$ctrl.selectedProject.metadata.uid" class="row">
+      <div ng-if="$ctrl.selectedProject.metadata.uid" class="col-sm-8 col-sm-offset-4">
+        <!-- TODO: Preserve entered form values. -->
+        To configure additional template parameters, view
```

Comment:
there are not necessarily additional parameters to configure though.  In some cases labels may be the only thing you get out of going to advanced.